docs/source/index.rst

```diff
@@ -1,22 +1,33 @@
-.. JupyterHub documentation master file, created by
-   sphinx-quickstart on Mon Jan  4 16:31:09 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 JupyterHub
 ==========
 
-.. note:: This is the official documentation for JupyterHub. This project is
-          under active development.
-
-JupyterHub is a multi-user server that manages and proxies multiple instances
-of the single-user Jupyter notebook server.
-
-Three actors:
-
-* multi-user Hub (tornado process)
-* `configurable http proxy <https://github.com/jupyter/configurable-http-proxy>`_ (node-http-proxy)
-* multiple single-user IPython notebook servers (Python/IPython/tornado)
+JupyterHub is a server that gives multiple users access to Jupyter notebooks,
+running an independent Jupyter notebook server for each user.
+
+To use JupyterHub, you need a Unix server (typically Linux) running
+somewhere that is accessible to your team on the network. It might be on an
+internal network at your organisation, or it can run on the public internet (in
+which case, take care with `security <getting-started.html#security>`__). It
+runs as an HTTP server. You can get users to go directly to the IP address of
+the server in their browser, or provide it with a domain name.
+
+There are different :doc:`authenticators <authenticators>` which control access
+to JupyterHub. The default one (pam) uses the user accounts on the server where
+JupyterHub is running. If you use this, you will need to create a user account
+on the system for each user on your team. Using other authenticators, you can
+allow users to sign in with e.g. a Github account, or with any single-sign-on
+system your organisation has.
+
+Then there are :doc:`spawners <spawners>`, which control how JupyterHub starts
```

Comment:
~~Then there are~~ Next, :doc:`spawners <spawners>`~~, which~~ control how JupyterHub starts the individual notebook servers for ~~individual~~ each user~~s~~.